Userland/Utilities/chown.cpp

```diff
@@ -16,19 +17,32 @@
 #include <sys/stat.h>
 #include <unistd.h>
 
+ErrorOr<int> change_owner(String path, uid_t new_uid, gid_t new_gid, bool dont_follow_symlinks);
+
+ErrorOr<int> change_owner(String path, uid_t new_uid, gid_t new_gid, bool dont_follow_symlinks)
```

Comment:
What is the need for the function template above the function itself?

Reply:
Without it, it throws a missing declaration error

```
../../Userland/Utilities/chown.cpp:20:14: error: no previous declaration for 'AK::ErrorOr<int> change_owner(AK::String, uid_t, gid_t, bool)' [-Werror=missing-declarations]
   20 | ErrorOr<int> change_owner(String path, uid_t new_uid, gid_t new_gid, bool dont_follow_symlinks)
```

Reply:
It definitely shouldn't be complaining about that, but I don't know enough about the compiler to see why it does in this case. We have loads of other tools that define utility functions without declaring them first.

Reply:
It doesn't complain if the function is marked `static`.

Reply:
@timschumi this is weird indeed. I never had this problem

@BertalanD  Ok, I will change to `static` then